source/frontend/src/features/properties/map/acquisition/AcquisitionContainer.tsx

```diff
@@ -1,28 +1,19 @@
-import { ReactComponent as RealEstateAgent } from 'assets/images/real-estate-agent.svg';
-import { GenericModal } from 'components/common/GenericModal';
 import { useMapSearch } from 'components/maps/hooks/useMapSearch';
 import LoadingBackdrop from 'components/maps/leaflet/LoadingBackdrop/LoadingBackdrop';
 import { FileTypes } from 'constants/index';
-import FileLayout from 'features/mapSideBar/layout/FileLayout';
-import MapSideBarLayout from 'features/mapSideBar/layout/MapSideBarLayout';
 import { FormikProps } from 'formik';
 import { Api_AcquisitionFile } from 'models/api/AcquisitionFile';
-import React, { useCallback, useContext, useEffect, useReducer, useRef, useState } from 'react';
-import styled from 'styled-components';
-import { getFilePropertyName } from 'utils/mapPropertyUtils';
+import React, { useCallback, useContext, useEffect, useReducer, useRef } from 'react';
 
 import { SideBarContext } from '../context/sidebarContext';
-import SidebarFooter from '../shared/SidebarFooter';
-import { UpdateProperties } from '../shared/update/properties/UpdateProperties';
-import { AcquisitionHeader } from './common/AcquisitionHeader';
-import AcquisitionMenu from './common/AcquisitionMenu';
+import { IAcquisitionViewProps } from './AcquisitionView';
 import { EditFormNames } from './EditFormNames';
 import { useAcquisitionProvider } from './hooks/useAcquisitionProvider';
-import ViewSelector from './ViewSelector';
 
 export interface IAcquisitionContainerProps {
   acquisitionFileId: number;
   onClose: () => void;
+  View: React.FunctionComponent<React.PropsWithChildren<IAcquisitionViewProps>>;
```

Comment:
Thanks for refactoring this container into the new dependency injection pattern!